wger/manager/templates/workout/view.html

```diff
@@ -219,12 +262,22 @@ <h4 class="modal-title">{% trans "Download as PDF" %}</h4>
         {% trans "Options" %}
     </button>
     <ul class="dropdown-menu">
+    {% if workout.canonical_representation.day_list %}
         <li>
-            <a data-toggle="modal" data-target="#download-pdf-popup">
+            <a data-toggle="modal" data-target="#export-workout-popup">
```

Comment:
Why did you change the target-data here? don't you think this might affect other areas of the code?

Reply:
It does not affect other areas of the code.

Have you tried testing the feature locally? That might help you understand why the change here was necessary.